### ISSUE TITLE
home, about, help and contact menu does not meet color contrast accessibility requirements

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6745]
### Descriptive summary

On dev.nurax.samvera.org (Hyrax 5.0.0), the top menu containing the Home, About, Help and Contact links does not meet WCAG AA minimum color contrast requirements

### Steps to reproduce the behavior in User Interface (UI)

1. In Chrome, download the Site Improve browser extension
2. Go to dev.nurax.samvera.org and run the Site Improve browser extension

![color_contrast_minimum](https://github.com/samvera/hyrax/assets/32644254/487c8de1-f792-46fd-a8d0-52703ecf744d)

### Rationale

Hyrax (and nurax) should be accessible to users out of the box.

**Comment by rkuehn-uofl** [comment]
Recommend removing "navbar-light' from nav class (~/hyrax/hyrax-v5.0.1/hyrax501/app/views/_controls.html.erb line #1) to meet color contrast requirement.
A Bootstrap override might be desired to override standard blue hover color.

**Comment by aahurford** [comment]
tested in dev and pg, no longer showing as an issue for home, about, help and contact nav bar (still an issue with depositor and keywords though)

![Image](https://github.com/user-attachments/assets/12e51165-bf45-445c-88f2-e73448acad2a)

**Comment by rjkati** [comment]
Depositor and keywords should be fixed by #6746. We'll address any lingering issues in that ticket.
